[TASK] Make it 9 compatible

ruudsilvrants opened the pull request:
TODO
Override tables with local table copy.

ruudsilvrants committed:
[TASK] Set version on 1.0.0

ruudsilvrants committed:
[TASK] Code optimalization and switch literal array syntax

ruudsilvrants committed:
[MERGE] Merge remote-tracking branch 'upstream/master'

ruudsilvrants committed:
[TASK] Update symfony process due removal of processBuilder

ruudsilvrants committed:
[TASK] Update TYPO3 database query for getting Storages

ruudsilvrants committed:
[TASK] Update to TYPO3 doctrine queries

ruudsilvrants committed:
[TASK] Remove debug statement

ruudsilvrants committed:
[TASK] Update version constraints

fsaris commented on Classes/Command/BackupCommandController.php:
```diff
@@ -302,28 +308,48 @@ protected function legacyRestore($tmpFolder, $fileBackups) {
+        /** @var Connection $connection */
+        $connection = GeneralUtility::makeInstance(ConnectionPool::class)->getConnectionForTable('fe_users');
+        try {
+            $connection->exec('UPDATE fe_users SET username = MD5(username), password = MD5(password), email = CONCAT(LEFT(UUID(), 8), "@beech.it") WHERE email NOT LIKE "%@beech.it"');
```
Why not use the query builder here to?

ruudsilvrants replied:
The MD5 could not be used. therefore this can be ignored

ruudsilvrants committed:
[TASK] Update tables with local fields by query
The queryBuilder does not except a join for update queries, therefore make a manual query
